Задания с 14.09.2022/Task1/Task1.Tests/ILazyTests.cs

```diff
@@ -0,0 +1,88 @@
+using System.Diagnostics;
+
+namespace ILazy.Tests;
+
+public class ILazyTests
+{
+    private static Stopwatch time = new();
+
+    [TestCaseSource(nameof(LazyEvaluationsForStandardCase))]
+    public void StandardCaseTest(ILazy<int> realization)
+    {
+        time.Restart();
+        var firstValue = realization.Get();
+        time.Stop();
+        var firstGetCallDuration = (float) time.Elapsed.TotalSeconds;
```

Comment:
```suggestion
        var firstGetCallDuration = (float)time.Elapsed.TotalSeconds;
```
И ниже. После оператора каста пробел обычно не ставится